android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/MultiSelectListFactory.java

```diff
@@ -53,7 +55,7 @@
     public String currentAdapterKey;
     public Context context;
     private JsonFormFragment jsonFormFragment;
-    private HashMap<String, MultiSelectListAccessory> multiSelectListAccessoryHashMap = new HashMap<>();
+    private static HashMap<String, MultiSelectListAccessory> multiSelectListAccessoryHashMap = new HashMap<>();
```

Comment:
what would happen if there is 2 multiselect in one form? given multiSelectListAccessoryHashMap is static

Reply:
@Andati - an adapterKey is used to get ItemList as well as the ListOfSelected items, so it will use the key of the defined multiselect for each. I have tested this with two MultiSelect in a single form and seems to be working. I am also open if you have any suggestion.